diff --git a/neofetch b/neofetch
index 64fa042..4778b64 100755
--- a/neofetch
+++ b/neofetch
@@ -11029,21 +11029,26 @@ EOF
         "Venom"*)
             set_colors 8 4
             read -rd '' ascii_data <<'EOF'
-${c1}   :::::::          :::::::
-   mMMMMMMm        dMMMMMMm
-   /MMMMMMMo      +MMMMMMM/
-    yMMMMMMN      mMMMMMMy
-     NMMMMMMs    oMMMMMMm
-     +MMMMMMN:   NMMMMMM+
-      hMMMMMMy  sMMMMMMy
-      :NMMMMMM::NMMMMMN:
-       oMMMMMMyyMMMMMM+
-        dMMMMMMMMMMMMh
-        /MMMMMMMMMMMN:
-         sMMMMMMMMMMo
-          mMMMMMMMMd
-          +MMMMMMMN:
-            ::::::
+${c1}
+   `-`                                
+    -yys+/-`                          
+     `oyyyyy: /osyyyyso+:.            
+       /yyyyy+`+yyyyyyyyyys/.         
+       .-yyyyys.:+//+oyyyyyyyo.       
+     `oy/`oyyyyy/      ./syyyyy:      
+     syyys`:yyyyyo`       :yyyyy:     
+    /yyyyo  .syyyyy-       .yyyyy.    
+    yyyyy.    +yyyyy/       /yyyy/    
+   `yyyyy      :yyyyys`     -yyyyo    
+    yyyyy.      `syyyyy-    /yyyy/    
+    /yyyyo        /yyyyy+  .yyyyy.    
+     syyyys.       -yyyyys.:yyyy:     
+     `oyyyyyo-`     `oyyyyy:.sy:      
+       :syyyyyyso+/++`/yyyyyo``       
+         -oyyyyyyyyyyy-.syyyys.       
+            -/+osyyyyso.`+yyyyy/      
+                          .-/+syo`    
+                                `.    
 EOF
         ;;
 
