diff -Naur neofetch-6.0.0-orig/neofetch neofetch-6.0.0/neofetch
--- neofetch-6.0.0-orig/neofetch	2019-01-09 00:32:04.000000000 +0800
+++ neofetch-6.0.0/neofetch	2019-04-13 17:17:52.000000000 +0800
@@ -1332,6 +1332,7 @@
             has "eopkg"   && dir ${br_prefix}/var/lib/eopkg/package/*
             has "crew"    && dir ${br_prefix}/usr/local/etc/crew/meta/*.filelist
             has "pkgtool" && dir ${br_prefix}/var/log/packages/*
+            has "scratch" && dir ${br_prefix}/var/lib/scratchpkg/index/*/.pkginfo
             has "cave"    && dir ${br_prefix}/var/db/paludis/repositories/cross-installed/*/data/*/ \
                                  ${br_prefix}/var/db/paludis/repositories/installed/data/*/
             }
@@ -4812,6 +4813,25 @@
             -/osssossossso/-
 EOF
         ;;
+        "Venom"*)
+            set_colors 8 4
+            read -rd '' ascii_data <<'EOF'
+${c1}
+   mMMMMMMm        dMMMMMMm 
+   /MMMMMMMo      +MMMMMMM/ 
+    yMMMMMMN      mMMMMMMy  
+     NMMMMMMs    oMMMMMMm   
+     +MMMMMMN:   NMMMMMM+   
+      hMMMMMMy  sMMMMMMy    
+      :NMMMMMM::NMMMMMN:    
+       oMMMMMMyyMMMMMM+     
+        dMMMMMMMMMMMMh      
+        /MMMMMMMMMMMN:      
+         sMMMMMMMMMMo       
+          mMMMMMMMMd        
+          +MMMMMMMN:        
+EOF
+        ;;
 
         "alpine_small")
             set_colors 4 7
