diff --git a/examples/dmabuf-capture.c b/examples/dmabuf-capture.c
index 4d05b78babeac34eac314461caa9685597d01261..58bf641958aa0316cf340ffb6d50864b61cde2a0 100644
--- a/examples/dmabuf-capture.c
+++ b/examples/dmabuf-capture.c
@@ -1,4 +1,5 @@
 #define _POSIX_C_SOURCE 199309L
+#include <libavcodec/avcodec.h>
 #include <libavformat/avformat.h>
 #include <libavutil/display.h>
 #include <libavutil/hwcontext_drm.h>
@@ -619,12 +620,12 @@ static int init_encoding(struct capture_context *ctx) {
 	}
 
 	/* Find encoder */
-	AVCodec *out_codec = avcodec_find_encoder_by_name(ctx->encoder_name);
+	const AVCodec *out_codec = avcodec_find_encoder_by_name(ctx->encoder_name);
 	if (!out_codec) {
 		av_log(ctx, AV_LOG_ERROR, "Codec not found (not compiled in lavc?)!\n");
 		return AVERROR(EINVAL);
 	}
-	ctx->avf->oformat->video_codec = out_codec->id;
+	ctx->avf->oformat = av_guess_format(ctx->encoder_name, NULL, NULL);
 	ctx->is_software_encoder = !(out_codec->capabilities & AV_CODEC_CAP_HARDWARE);
 
 	ctx->avctx = avcodec_alloc_context3(out_codec);
