diff -urN slim_themes.orig/themes/bluer/slim.theme slim_themes/themes/bluer/slim.theme
--- slim_themes.orig/themes/bluer/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/bluer/slim.theme	2023-08-31 20:30:43.793060391 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/boxy/slim.theme slim_themes/themes/boxy/slim.theme
--- slim_themes.orig/themes/boxy/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/boxy/slim.theme	2023-08-31 20:31:02.097061137 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/bridge/slim.theme slim_themes/themes/bridge/slim.theme
--- slim_themes.orig/themes/bridge/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/bridge/slim.theme	2023-08-31 20:31:27.777062185 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/cayny/slim.theme slim_themes/themes/cayny/slim.theme
--- slim_themes.orig/themes/cayny/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/cayny/slim.theme	2023-08-31 20:32:03.265063632 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/darksome/slim.theme slim_themes/themes/darksome/slim.theme
--- slim_themes.orig/themes/darksome/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/darksome/slim.theme	2023-08-31 20:32:21.912064393 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/darky_pink/slim.theme slim_themes/themes/darky_pink/slim.theme
--- slim_themes.orig/themes/darky_pink/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/darky_pink/slim.theme	2023-08-31 20:32:35.377064942 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/desky/slim.theme slim_themes/themes/desky/slim.theme
--- slim_themes.orig/themes/desky/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/desky/slim.theme	2023-08-31 20:33:08.281066284 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/faded_city/slim.theme slim_themes/themes/faded_city/slim.theme
--- slim_themes.orig/themes/faded_city/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/faded_city/slim.theme	2023-08-31 20:33:25.841067000 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/flat/slim.theme slim_themes/themes/flat/slim.theme
--- slim_themes.orig/themes/flat/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/flat/slim.theme	2023-08-31 20:33:44.225067750 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/flat_green/slim.theme slim_themes/themes/flat_green/slim.theme
--- slim_themes.orig/themes/flat_green/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/flat_green/slim.theme	2023-08-31 20:34:02.660068501 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/greeny/slim.theme slim_themes/themes/greeny/slim.theme
--- slim_themes.orig/themes/greeny/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/greeny/slim.theme	2023-08-31 20:34:18.362069142 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/greeny_dark/slim.theme slim_themes/themes/greeny_dark/slim.theme
--- slim_themes.orig/themes/greeny_dark/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/greeny_dark/slim.theme	2023-08-31 20:34:39.305069996 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/hello/slim.theme slim_themes/themes/hello/slim.theme
--- slim_themes.orig/themes/hello/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/hello/slim.theme	2023-08-31 20:34:56.675070704 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/light_red/slim.theme slim_themes/themes/light_red/slim.theme
--- slim_themes.orig/themes/light_red/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/light_red/slim.theme	2023-08-31 20:35:17.289071545 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/material/slim.theme slim_themes/themes/material/slim.theme
--- slim_themes.orig/themes/material/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/material/slim.theme	2023-08-31 20:35:35.058072270 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/milk/slim.theme slim_themes/themes/milk/slim.theme
--- slim_themes.orig/themes/milk/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/milk/slim.theme	2023-08-31 20:35:54.242073052 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/milkyway/slim.theme slim_themes/themes/milkyway/slim.theme
--- slim_themes.orig/themes/milkyway/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/milkyway/slim.theme	2023-08-31 20:36:08.346073627 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/minimal/slim.theme slim_themes/themes/minimal/slim.theme
--- slim_themes.orig/themes/minimal/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/minimal/slim.theme	2023-08-31 20:36:32.594074616 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/overlay/slim.theme slim_themes/themes/overlay/slim.theme
--- slim_themes.orig/themes/overlay/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/overlay/slim.theme	2023-08-31 20:37:35.938077200 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/panda/slim.theme slim_themes/themes/panda/slim.theme
--- slim_themes.orig/themes/panda/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/panda/slim.theme	2023-08-31 20:37:51.466077833 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/pinker/slim.theme slim_themes/themes/pinker/slim.theme
--- slim_themes.orig/themes/pinker/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/pinker/slim.theme	2023-08-31 20:38:06.186078433 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/pot/slim.theme slim_themes/themes/pot/slim.theme
--- slim_themes.orig/themes/pot/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/pot/slim.theme	2023-08-31 20:38:32.533079508 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/sweet_flat/slim.theme slim_themes/themes/sweet_flat/slim.theme
--- slim_themes.orig/themes/sweet_flat/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/sweet_flat/slim.theme	2023-08-31 20:38:48.459080157 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/typogin/slim.theme slim_themes/themes/typogin/slim.theme
--- slim_themes.orig/themes/typogin/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/typogin/slim.theme	2023-08-31 20:39:13.278081169 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/wavy/slim.theme slim_themes/themes/wavy/slim.theme
--- slim_themes.orig/themes/wavy/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/wavy/slim.theme	2023-08-31 20:39:26.251081698 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/white/slim.theme slim_themes/themes/white/slim.theme
--- slim_themes.orig/themes/white/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/white/slim.theme	2023-08-31 20:39:41.948082339 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
diff -urN slim_themes.orig/themes/workspace/slim.theme slim_themes/themes/workspace/slim.theme
--- slim_themes.orig/themes/workspace/slim.theme	2023-08-21 08:13:09.000000000 +0200
+++ slim_themes/themes/workspace/slim.theme	2023-08-31 20:39:56.155082918 +0200
@@ -37,6 +37,6 @@
 
 # Authentication
 
-password_feedback_x     50%
-password_feedback_y     30%
-password_feedback_msg   Authentication failed
+passwd_feedback_x     50%
+passwd_feedback_y     30%
+passwd_feedback_msg   Authentication failed
