diff -Naur scratchpkg-9c41d50f45cdbb7a03a78f6640467b8bf60a428a-orig/pkgbuild scratchpkg-9c41d50f45cdbb7a03a78f6640467b8bf60a428a/pkgbuild
--- scratchpkg-9c41d50f45cdbb7a03a78f6640467b8bf60a428a-orig/pkgbuild	2020-01-11 15:29:36.080000000 +0800
+++ scratchpkg-9c41d50f45cdbb7a03a78f6640467b8bf60a428a/pkgbuild	2020-01-11 15:30:44.770000000 +0800
@@ -200,7 +200,7 @@
 	
 	pushd $SRC >/dev/null
 	
-	(set -e -x; build 2>&1) ||
+	(set -e -x; build 2>&1) 
 		
 	if [ $? != 0 ]; then
 		msgerr "Build '$name-$version-$release' failed."
