--- grub-2.12/util/grub.d/10_linux.in-ori	2023-12-22 12:16:58.386810228 +0100
+++ grub-2.12/util/grub.d/10_linux.in	2023-12-22 12:25:15.381835867 +0100
@@ -224,6 +224,7 @@
 
   initrd_real=
   for i in "initrd.img-${version}" "initrd-${version}.img" \
+	   "initrd-venom-*.img" \
 	   "initrd-${alt_version}.img.old" "initrd-${version}.gz" \
 	   "initrd-${alt_version}.gz.old" "initrd-${version}" \
 	   "initramfs-${version}.img" "initramfs-${alt_version}.img.old" \
