diff -Nur i3-4.16.1.orig/configure.ac i3-4.16.1/configure.ac
--- i3-4.16.1.orig/configure.ac	2019-01-27 23:45:19.000000000 +0800
+++ i3-4.16.1/configure.ac	2019-05-29 12:36:22.319997066 +0800
@@ -101,6 +101,8 @@
 PKG_CHECK_MODULES([YAJL], [yajl])
 PKG_CHECK_MODULES([LIBPCRE], [libpcre >= 8.10])
 PKG_CHECK_MODULES([PANGOCAIRO], [cairo >= 1.14.4 pangocairo])
+PKG_CHECK_MODULES([GLIB2], [glib-2.0])
+PKG_CHECK_MODULES([GOBJECT2], [gobject-2.0])
 
 # Checks for programs.
 AC_PROG_AWK
diff -Nur i3-4.16.1.orig/Makefile.am i3-4.16.1/Makefile.am
--- i3-4.16.1.orig/Makefile.am	2019-01-27 23:45:19.000000000 +0800
+++ i3-4.16.1/Makefile.am	2019-05-29 12:35:31.439997083 +0800
@@ -277,6 +277,8 @@
 
 libi3_CFLAGS = \
 	$(AM_CFLAGS) \
+	$(GLIB2_CFLAGS) \
+	$(GOBJECT2_CFLAGS) \
 	$(XCB_CFLAGS) \
 	$(XCB_UTIL_CFLAGS) \
 	$(XCB_UTIL_XRM_CFLAGS) \
@@ -285,6 +287,8 @@
 
 libi3_LIBS = \
 	$(top_builddir)/libi3.a \
+	$(GLIB2_LIBS) \
+	$(GOBJECT2_LIBS) \
 	$(XCB_LIBS) \
 	$(XCB_UTIL_LIBS) \
 	$(XCB_UTIL_XRM_LIBS) \
