diff -urN pangomm-2.42.2.orig/untracked/pango/pangomm/attrlist.cc pangomm-2.42.2/untracked/pango/pangomm/attrlist.cc
--- pangomm-2.42.2.orig/untracked/pango/pangomm/attrlist.cc	2020-12-13 08:11:13.689664100 -0600
+++ pangomm-2.42.2/untracked/pango/pangomm/attrlist.cc	2021-12-21 13:40:02.471631496 -0600
@@ -6,6 +6,7 @@
 #include <pangomm/attrlist.h>
 #include <pangomm/private/attrlist_p.h>
 
+#include <pango-1.0/pango/pango-markup.h>
 
 // -*- c++ -*-
 /* $Id: attrlist.ccg,v 1.3 2005/11/22 15:00:17 murrayc Exp $ */
