diff -Naur mate-sensors-applet-1.24.0-ori/plugins/i2c-proc/i2c-proc-plugin.c mate-sensors-applet-1.24.0/plugins/i2c-proc/i2c-proc-plugin.c
--- mate-sensors-applet-1.24.0-ori/plugins/i2c-proc/i2c-proc-plugin.c	2020-04-08 13:24:52.166074751 +0800
+++ mate-sensors-applet-1.24.0/plugins/i2c-proc/i2c-proc-plugin.c	2020-04-08 13:25:38.886074533 +0800
@@ -31,6 +31,7 @@
 #include <glib.h>
 #include <glib/gi18n.h>
 #include "i2c-proc-plugin.h"
+#include <locale.h>
 
 const gchar *plugin_name = "i2c-proc";
 
