diff --git a/pkgadd b/pkgadd
index 7823f4a..f0a8229 100755
--- a/pkgadd
+++ b/pkgadd
@@ -276,6 +276,7 @@ done
 if [ "$UPGRADE_PKG" ] || [ "$REINSTALL_PKG" ]; then
 	tail -n+2 "$ROOT_DIR/$PKGDB_DIR/$name" | tac | while read -r line; do
 		case $line in
+			*/[) ;;
 			*/) grep "^$line$" $ROOT_DIR/$PKGDB_DIR/* "$TMP_PKGINSTALL" 2>/dev/null | grep -qv "$ROOT_DIR/$PKGDB_DIR/$name" || rmdir $([ "$VERBOSE_INSTALL" ] && echo -v) "$ROOT_DIR/$line";;
 			 *) grep -q "^$line$" "$TMP_PKGINSTALL" || rm $([ "$VERBOSE_INSTALL" ] && echo -v) "$ROOT_DIR/$line";;
 		esac
