--- /var/cache/scratchpkg/work/fim/src/fim-0.5-rc3/src/fim.cpp	2017-07-29 09:04:17.000000000 +0000
+++ ./src/fim.cpp	2021-12-31 02:29:38.469529248 +0000
@@ -343,8 +343,8 @@
  .P
  .nf
  Shift+D         delete image
- R               rotate 90° clockwise
- L               rotate 90° counter-clock wise
+ R               rotate 90 deg clockwise
+ L               rotate 90 deg counter-clock wise
  .fi
  .P
  The delete function actually wants a capital letter 'D', thus you have
